### DISCUSSION TITLE
Using as a standalone install

### DISCUSSION THREAD

**Discussion by mrjanto** [Esri/arcgis-powershell-dsc#594] [Q&A] [answered]
I'm new to DSC, and I'm wondering what happens when I execute one of these scripts.  If I upgrade ArcGIS Server to 10.9.1 by executing a PowerShell script with a configuration, does it get added to the LCM for DSC to periodically check the software after that?  If I want to upgrade to 11.3 after that and I run it myself rather than through a DSC script, will DSC try to reset it to 10.9.1?  If so, how do I remove it from LCM?
Also, when I execute an upgrade, where does it store the MOF?  If I only want to run the upgrade once, I'd like to remove the MOF file if it has passwords in it.

**Comment by cameronkroeker** [comment, marked as answer]
Hi @mrjanto,

It is recommended to set the Location Configuration Manager (LCM) ConfigurationMode to `ApplyOnly`:

https://github.com/Esri/arcgis-powershell-dsc/wiki/Getting-Started#system-pre-requisites

The ArcGIS module always deletes the MOFs. You can also use Encrypted MOFs:

https://github.com/Esri/arcgis-powershell-dsc/wiki/Password-Files-and-Encrypted-Deployments#create-encrypted-mof-files

Thanks,
Cameron K.
